Mayhemfile

```diff
@@ -0,0 +1,5 @@
+project: jbgreenh/cpp-base64

```

Comment:
Please genericize the project name and update the target name after changing the fuzz target.